std_msgs/Header header
float32 x # m
float32 y # m
float32 z # m
float32 vx
float32 vy
float32 vz
float32 ax
float32 ay
float32 az
float32 yaw # rad
float32 yaw_dot